internal/pkg/keybindings/listhandlers.go

```diff
@@ -432,13 +435,12 @@ func (h ListUpdateHandler) getEditorConfig() (config.EditorConfig, error) {
 		return userConfig.Editor, nil
```

Comment:
future: One odd thing I hit was when I set "vim" but didn't have "vim" installed only "vi" the edit panel will hang. Maybe worth pulling out an issue to do some validation on the executable being in the path and throwing a more human error if it's not but it's not a blocker to this work for me.

Reply:
Ah, good spot - I wasn't reinstating the alt buffer if there is an error launching the editor!

Have added a commit to fix this